gooddata-server-oauth2-autoconfigure/src/main/kotlin/AuthenticationUtils.kt

```diff
@@ -233,7 +233,8 @@ private fun ClientRegistration.resolveSupportedScopes() =
 
 private fun ClientRegistration.Builder.withScopes(supportedScopes: Scope?): ClientRegistration.Builder {
     // in the future, we could check mandatory scopes against the supported ones
-    val mandatoryScopes = listOf(OIDCScopeValue.OPENID, OIDCScopeValue.PROFILE).map(Scope.Value::getValue)
+    val mandatoryScopes =
+        listOf(OIDCScopeValue.OPENID, OIDCScopeValue.PROFILE, OIDCScopeValue.EMAIL).map(Scope.Value::getValue)
```

Comment:
Can not be the email optional?

Reply:
Makes sense. I've added `gd_user_groups` scope and configured both `email` & `gd_users_groups `scopes as optional.